pkg/ccl/backupccl/backup_test.go

```diff
@@ -9494,6 +9508,11 @@ func TestExcludeDataFromBackupAndRestore(t *testing.T) {
 	restoreDB.Exec(t, `RESTORE DATABASE data FROM LATEST IN $1`, localFoo)
 	require.Len(t, restoreDB.QueryStr(t, `SELECT * FROM data.foo`), 0)
 	require.Len(t, restoreDB.QueryStr(t, `SELECT * FROM data.bar`), 10)
+
+	before := atomic.LoadInt64(&exportReqsAtomic)
```

Comment:
so at this point data.foo is going to be empty as asserted above. Can we add a few rows to it before attempt a backup.

Also s/`BACKUP TO`/`BACKUP INTO`/g